_includes/manuals/1.18/3.5.2_configuration_options.md

```diff
@@ -416,8 +416,8 @@ Default: foreman_organization
 
 ##### outofsync_interval
 
-Duration in minutes after the Puppet interval for servers to be classed as out of sync.  This is added to the puppet_interval setting and used in pages such as the host list and dashboard.
-Default: 5
+Duration in minutes after the servers classed as out of sync.
```

Comment:
I think it might be good to describe under which conditions this applies. IIRC the host gets out of sync if the most recent report is older than $out_of_sync_interval minutes and this report origin is unknown. Or did I mixed that up?